src/main/java/com/elmakers/mine/bukkit/action/builtin/AnimateAction.java

```diff
@@ -22,774 +20,4 @@ public SpellResult step(CastContext context) {
 	};
 
 	public static boolean DEBUG = false;
-
```

Comment:
Why the big chop here?

This was admittedly a big TODO that never got done, just curious why it had to be stripped.